pkg/workloadmeta/types.go

```diff
@@ -57,6 +57,10 @@ type Store interface {
 	// to all entities with kind KindContainer.
 	ListContainers() ([]*Container, error)
```

Comment:
I don't think we need to keep that. Passing a nil filter should be enough

Reply:
Right, I thought about that, but for callers that just want to list everything it's easier than calling with something like `(nil)` or `(GetAll)`.